src/main/java/net/sf/jsqlparser/util/CNFexpression/MultiAndExpression.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * #%L
+ * JSQLParser library
+ * %%
+ * Copyright (C) 2004 - 2017 JSQLParser
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 2.1 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Lesser Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Lesser Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ * #L%
+ */
+package net.sf.jsqlparser.util.CNFexpression;
+
+import java.util.List;
+
+import net.sf.jsqlparser.expression.Expression;
+
+/**
+ * This helper class is mainly used for handling the CNF conversion.
```

Comment:
Maybe a better description could be "Groups expressions that should be ANDED together for CNF Conversion" (same with the MultiOrExpression